flink-table/flink-table-code-splitter/src/main/resources/META-INF/NOTICE

```diff
@@ -7,7 +7,7 @@ The Apache Software Foundation (http://www.apache.org/).
 This project bundles the following dependencies under the BSD 3-clause license.
 See bundled license files for details.
 
-- antlr:4.7
+- org.antlr:antlr4-runtime:4.7
```

Comment:
I took a look to https://github.com/antlr/grammars-v4/blob/master/java/java/pom.xml#L4 for `JavaLexer.g4`.
There should be two.

Reply:
Grammar files are also mentioned in this NOTICE file. This `org.antlr.grammars:java` does not seem to exist in maven repository, so we have to copy the grammar files into Flink.